README.md

```diff
@@ -29,7 +29,7 @@ This is pre release and there will be major changes to this before its final rel
 - :master_key: The Octopus Deploy Server master key for encryption, leave blank to generate one at creation
 - :node_name: The Octopus Deploy Server Node Name, will default to machine hostname
 - :create_database: Whether Octopus Deploy Server should create the database with the connection string provided (Defaults to false)
-- :admin_user: A default admin in AD for the Octopus Deploy Server to create
+- :admin_user: A default admin user for Octopus Deploy Server to create. Requires machine to be joined to active directory, and `admin_user` must be an AD user
```

Comment:
Not sure if this should be revisited because now you can have authentication providers

Reply:
Agreed, possibly this could be improved more. It took me quite a while to figure out why I couldn't add local users. This update was just to prevent someone from falling into the same pit as me. I'm ok to close until auth providers are implemented. ( I hadn't heard of those until now)

Reply:
I think this documentation makes sense so I will merge.  Also going to create a new issue to track auth providers.